.gitignore

```diff
@@ -1,6 +1,7 @@
 *.userprefs
 bin
 Configuration.Override.props
+Configuration.OperatingSystem.props
```

Comment:
Should this be `Configuration.BuildOS.props`? I can see some confusion about the OS being the *source*/*host* OS or the *target* OS, but maybe I'm overthinking it?